src/planner.tsx

```diff
@@ -118,7 +118,7 @@ function planSetContact(context: Plan, privateContact: Contact): Plan {
   const event = finalizeEvent(
     {
       kind: KIND_REPUTATIONS,
-      pubkey: getPublicKey(context.user.privateKey),
+      pubkey: context.user.publicKey,
```

Comment:
getPublicKey returns a publicKey and not an `npub`. If you look how we create users in

 ```function userFromPrivateKey(privateKey: string): KeyPair {
  const key = hexToBytes(privateKey);
  const publicKey = getPublicKey(key) as PublicKey;
  return {
    publicKey,
    privateKey: key,
  };
}
```

we do exactly the same